aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -8,7 +8,12 @@ module aptos_framework::aptos_coin {
     use std::option::{Self, Option};
 
     use aptos_framework::coin::{Self, BurnCapability, MintCapability};
+    use aptos_framework::fungible_asset::{generate_mint_ref, generate_transfer_ref, generate_burn_ref};
```

Comment:
we really ought not import functions :bow:

Reply:
I would fix my part~~~